recipes/backward-cpp/all/patches/backward-cpp-1.5-add-iterator-include.patch

```diff
@@ -1,10 +1,10 @@
 --- backward.hpp.old	2020-05-05 20:37:23.473359490 +0200
```

Comment:
😆

Reply:
This patch is already in upstream:
https://github.com/bombela/backward-cpp/blame/bd9389453a462c85a70f03718840ca0897c9088b/backward.hpp#L94
(3 months ago)